app/models/marc_fields/issn.rb

```diff
@@ -1,15 +1,19 @@
 ##
-# A class to handle MARC 020a + 020z field logic
-# https://www.loc.gov/marc/bibliographic/bd020.html
+# A class to handle MARC 022a + 022z field logic
+# https://www.loc.gov/marc/bibliographic/bd022.html
 class Issn < MarcField
   def values
-    super.map(&:strip).select { |v| v =~ issn_pattern }
+    return [] if marc.blank?
+
+    @values ||= extracted_fields.flat_map do |field, subfields|
+      subfields.map(&:value).map(&:strip).select { |v| v =~ issn_pattern }
+    end.compact
```

Comment:
Can you say more about how the ISSN changes relate to the bound_with changes? Or maybe they don't.